src/backend/base/langflow/services/auth/utils.py

```
@@ -162,56 +163,83 @@ async def get_current_user_by_jwt(
-        logger.error("Secret key is not set in settings.")
-        raise HTTPException(
-            status_code=status.HTTP_401_UNAUTHORIZED,
-            # Careful not to leak sensitive information
-            detail="Authentication failure: Verify authentication settings.",
-            headers={"WWW-Authenticate": "Bearer"},
-        )
+    if settings_service.auth_settings.CLERK_AUTH_ENABLED:
+        try:
+            claims = verify_clerk_token(token)
+            clerk_user_id = get_clerk_user_id(claims)
+            email = get_clerk_user_email(claims)
```

Ruff (0.11.9)

Local variable `email` is assigned to but never used

Remove assignment to unused variable `email`

(F841)